Failure to find matplotlibrc when testing installed distribution

During wheel testing:

```
_______________________ test_if_rctemplate_is_up_to_date _______________________
[gw0] linux -- Python 3.5.3 /venv/bin/python3.5

    def test_if_rctemplate_is_up_to_date():
        # This tests if the matplotlibrc.template file
        # contains all valid rcParams.
        dep1 = mpl._all_deprecated
        dep2 = mpl._deprecated_set
        deprecated = list(dep1.union(dep2))
        #print(deprecated)
        path_to_rc = mpl.matplotlib_fname()
>       with open(path_to_rc, "r") as f:
E       FileNotFoundError: [Errno 2] No such file or directory: '/venv/lib/python3.5/site-packages/matplotlib/mpl-data/matplotlibrc'

```

See full log available via https://travis-ci.org/MacPython/matplotlib-wheels/jobs/350816982

Is the matplotlibrc being correctly bundled / registered for install?

--- a/setupext.py
+++ b/setupext.py
@@ -669,6 +669,7 @@ class Matplotlib(SetupPackage):
         return {
             'matplotlib':
             [
+                'mpl-data/matplotlibrc',
                 *iter_dir('mpl-data/fonts'),
                 *iter_dir('mpl-data/images'),
                 *iter_dir('mpl-data/stylelib'),

